webapp/channels/src/components/widgets/separator/basic-separator.tsx

```diff
@@ -9,21 +9,20 @@ type Props = {
     children?: ReactNode;
 };
 
-export default class BasicSeparator extends React.PureComponent<Props> {
-    public render() {
-        const {children} = this.props;
-        return (
-            <div
-                data-testid='basicSeparator'
-                className='Separator BasicSeparator'
-            >
-                <hr className='separator__hr'/>
-                {children && (
-                    <div className='separator__text'>
-                        {children}
-                    </div>
-                )}
-            </div>
-        );
-    }
+function BasicSeparator({children}: Props) {
```

Comment:
Can you make this a arrow function? I think there is not much difference, but I want to keep as much as possible the coherence between different parts of the code, and that means all function components to be arrow functions :sweat_smile:

Reply:
fixed with [f6202a0](https://github.com/mattermost/mattermost/pull/24788/commits/f6202a0d91cf59038bbbd01a47272cfda7b20a99)